### ISSUE TITLE
No connecting to 1 RDP server

### ISSUE THREAD

**Issue by ComputingFroggy** [FreeRDP/Remmina#1079]
Hello,
I am using _remmina_ to connect via _RDP_ so some _Windows_ servers (and also, using _VNC_ to connect to customers workstations ... but that's not relevant for that issue).

On most servers, I've been using  _remmina_ successfully and happily.
But on that new server (installed by some other person), I get an error message each time I am trying to connect to that server: "Failure to connect to RDP server" (it might not be the exact wording as I am translating the error message from French).
I can access to other _Windows 2012_ servers with _remmina_ (thankfully). I can connect to that same new server from Windows (running _VirtualBox_ on my Linux machine or from another server).

Is that some configuration on the server preventing Linux connections ?
Or is there something in the _Windows 2012_ server configuration that prevents connection to that server ?

Let me know if I can help : this is really annoying although I am not completely locked as I can still connect through a _Windows_ connection.

Please give us some details about the systems you are using:
* Client (OS name and version): Xubuntu 16.04
* Remmina version (remmina --version): 1.2.0-rcgit-17
* Desktop environment (GNOME, Unity, KDE, ..): XFCE
* Connecting to (OS and version): Windows Server 2012
* Connecting via (RDP, VNC, ...): RDP
* RDP issues only, freerdp version (xfreerdp --version): xfreerdp is not installed ! ? ! ?

**Comment by weberhofer** [comment]
@ComputingFroggy: Can this be related to #1000?

**Comment by ComputingFroggy** [comment]
I have just realised: I am not connecting to _Windows 2012 R2 server_ but to _Windows 2008 R2 server_ !
I was told it was 2012 and I did not check ... but after playing around on the server, I was in doubt, and I checked : it is indeed _Windows 2008 R2_ !

@weberhofer : I am not sure whether this is related to #1000 ... I don't have local access !

**Comment by weberhofer** [comment]
In that case I don't think its related to #1000. Do you see any information when you open the debug-window in Remmina or when you start it via the console?

**Comment by ComputingFroggy** [comment]
No: no extra information in the debug nor the terminal (when starting _remmina_ from the terminal).

**Comment by ComputingFroggy** [comment]
I have found a work around !
Just to precise: I am connecting via a RDP connection using a server name which is managed by _dynalias_ (xxxx.dynalias.net). 
If, instead of connecting to xxxx.dynalias.net, I connect to the (long) domain name used, that works !

So it seems it's a compatibility problem with _dynalias_ services. Keep in mind that I can connect without any problem when connecting using "Connect to remote desktop" from _Windows_.

**Comment by antenore** [comment]
WARNING:
The Remmina project is moving to gitlab.com, in order to keep receiving notification about this issue,
you should have a gitlab account and link it to github.
The migration will happen in the next days.
Thanks for your support and understanding.

cc @ComputingFroggy @weberhofer
